lms/djangoapps/certificates/admin.py

```diff
@@ -46,8 +47,16 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     list_display = ('description', '__unicode__')
 
 
+class GeneratedCertificateAdmin(admin.ModelAdmin):
+    """
+    Django admin customizations for GeneratedCertificae model
+    """
+    list_display = ('id', 'course_id', 'mode', 'user')
```

Comment:
This is fine for the list display. The other fields will show on the detail page.